### DISCUSSION TITLE
Modify the PublicKeyToken of cefsharp dlls works for C# projects but not VCX projects ( CefSharp.Core.Runtime and CefSharp.BrowserSubprocess.Core )

### DISCUSSION THREAD

**Discussion by Dileepreddyj** [cefsharp/CefSharp#4746] [General]
For security purposes, I am attempting to modify the PublicKeyToken for the CefSharp DLLs by replacing the existing cefsharp.snk file with my own .snk file. While I observe the change correctly reflected in all C# projects, the same is not true for the VCX projects ( CefSharp.Core.Runtime and CefSharp.BrowserSubprocess.Core ). I have verified the Key File settings for these projects under "Configuration Properties" -> "Linker" -> "Advanced" section and confirmed the presence of the new .snk file. However, the PublicKeyToken values in these two DLLs persist from the original CefSharp.snk file, not the new .snk file. Kindly advise if there are other locations I need to modify to ensure the new PublicKeyToken is applied to these VCX project DLLs.

**Comment by amaitland** [comment]
If you open the `.vcxproj` files in a text editor you'll see there's multiple places that `CefSharp.snk` is used (different build permutations). You'd need to update them all.
